### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

**Comment by ottok** [comment]
Thanks for contributing to rdiff-backup!

As you might have noticed, the project has been stale for some years now. We intend to revitalize it now. I hope you can be involved in the effort!

The biggest ongoing change is the rewrite to Python 3 (in https://github.com/rdiff-backup/rdiff-backup/pull/40) and once that is merged we will have to ask you to rebase this contribution on the new Python 3 code base. Thanks for understanding and your help!

**Comment by jsoref** [comment]
That's fine. I do these tasks asynchronously, and my tooling supports incremental checking (although I'd rather not do it too frequently). Ping me again when you're ready. Rebasing isn't a problem.

**Comment by ottok** [comment]
Hello!

Most of the code base is now rewritten in Python 3. We also have a CI pipeline now. Could you please rebase this PR on the latest upstream master?

We would also be very grateful if you want to contribute to the project fixing other issues as well besides this PR.

**Comment by jsoref** [comment]
This was a purely automated rebase:
```sh
for a in $(hg log -T '{rev} ' -r 'spelling-old%master'); do
  X="$(MODE=rs convertpatch-to -c$a)"; sh -c "$X" 2>&1
done
```

Note:
* `subtract_indices`, `add_indicies`, and `get_indicies` are referenced but don't appear to exist
* `indicies` (when used as a variable) is a typo (when used as part of a method, it's mostly a reference to a removed function as mentioned above)

Because there's no human involved in this, I'm just going to talk to your CI until it stops complaining. `convertpatch-to` isn't smart enough to understand how to deal w/ word boundaries, so if there was a case where `subproces` was fixed as `subprocess` and there were cases where `subprocess` was already correctly spelling, it will fix the former and corrupt the latter. I'll look into improving that later (it's possible, but takes time, and for the time being, i'd rather rely on 👀 and 🤖 ).

File: CHANGELOG [hunk 1]
```diff
@@ -1582,7 +1582,7 @@ New in v0.5.2 (2002/03/02)
 Fixed bugs which made rdiff-backup try to preserve mod times when it
 wasn't necessary, and exit instead of warning when it wasn't being run
 as root and found a file it didn't own.  (Reported by Alberto
-Accomazi.)
+Accomazzi.)
```
Comment (ericzolf) [inline CHANGELOG]: Please revert change, not sure we want to fix people's name ;-)

Comment (ericzolf) [reply CHANGELOG]: Ah well, it looks like even this correction is correct 8-/

**Pull request merged**